views/assets/styles/components/_booking.scss

```diff
@@ -0,0 +1,213 @@
+.booking {
```

Comment:
INFO - We wouldn't do booking or Course display in management.#
INFO - Discite Components etc